app/serializers/admin_order_serializer.rb

```diff
@@ -0,0 +1,30 @@
+class AdminOrderSerializer
+  include JSONAPI::Serializer
+
+  attributes :total_amount, :status, :created_at
+
+  def self.record_type
+    "order"
+  end
+
+  attribute :user_info do |order|
+    {
+      name: order.user.name,
+      email: order.user.email
+    }
+  end
+
+  attribute :order_items_info do |order|
+    order.order_items.map do |item|
```

Comment:
you should use associations for this https://github.com/okuramasafumi/alba?tab=readme-ov-file#serialization-with-associations